configure.ac

```diff
@@ -109,6 +109,9 @@ AC_CHECK_HEADERS(
 
 PDNS_CHECK_READLINE
 PDNS_CHECK_RAGEL
+# We need curl to retrieve the effective tld list
+PDNS_CHECK_CURL_PROGRAM
```

Comment:
I'd rather see `make dist` just die without `curl`, than forcing this dep on all auth users.